hail/python/hailtop/hailctl/dataproc/stop.py

```diff
@@ -5,14 +5,22 @@ def init_parser(parser):
     parser.add_argument('name', type=str, help='Cluster name.')
     parser.add_argument('--async', action='store_true', dest='asink',
                         help="Do not wait for cluster deletion.")
+    parser.add_argument('--region', help='Region.', required=True)
     parser.add_argument('--dry-run', action='store_true',
                         help="Print gcloud dataproc command, but don't run it.")
 
 
 def main(args, pass_through_args):
     print("Stopping cluster '{}'...".format(args.name))
 
-    cmd = ['dataproc', 'clusters', 'delete', '--quiet', args.name]
+    cmd = [
+        'dataproc', 
+        'clusters', 
+        'delete', 
+        '--region={}'.format(args.region), 
```

Comment:
Why not an f-string here?

```suggestion
       f'--region={args.region}'
```

Reply:
Totally agree about f-strings in general, but this one was just for consistency with `submit.py`